index.js

```diff
@@ -62,6 +49,10 @@ class MongoModels {
 
     static async connect(connection, options = {}, name = 'default') {
 
+        if (!options.hasOwnProperty('useNewUrlParser')) {
```

Comment:
Consider using `hoek` for default option values.

Reply:
Also why do we want to default this option to `true`? Why not use the default behavior from the underlying driver?

Reply:
Changed to use Hoek. Without being set to true you get a deprecation message: "DeprecationWarning: current URL string parser is deprecated, and will be removed in a future version. To use the new parser, pass option { useNewUrlParser: true } to MongoClient.connect."